rotary/rules/rotary.smk

```diff
@@ -8,9 +8,11 @@ import itertools
 from snakemake.utils import min_version
 
 from rotary.sample import parse_sample_tsv
-from rotary.utils import symlink_or_compress
+from rotary.utils import symlink_or_compress, get_contamination_reference_files
 
 
+PHIX_GENOME_ACCESSION = "GCF_000819615.1"
```

Comment:
@jmtsuji I think we should eventually move these versions to a separate configuration file. The database version list is starting to get quite long.

Reply:
I agree -- I was thinking the same thing. Will convert to issue.

Reply:
Done -- #78